### ISSUE TITLE
Store submission for Azure DevOps

### ISSUE THREAD

**Issue by JanRajnoha** [microsoft/store-submission#5]
Is any official task for publishing into Microsoft Store on Azure DevOps and if no, will you consider one? On Azure DevOps is one, which is 2 years without commit and last release according to GitHub is 4 years old.

**Comment by azchohfi** [comment]
We are working on publishing this exact same task to Azure DevOps asap. It will have the same scope of being targeted for Win32 apps (.exe and .msi).

**Comment by JanRajnoha** [comment]
Thank you very much for comment. I can't wait to happen.

**Comment by azchohfi** [comment]
I'll leave this open so we can close it when we ship it, so you are notified.

**Comment by azchohfi** [comment]
@JanRajnoha can you try to use the Microsoft Store Dev CLI?
https://github.com/microsoft/msstore-cli

**Comment by JanRajnoha** [comment]
@azchohfi I will check Azure DevOps capabilities (VMs, jobs, ...) and try it

**Comment by Panda-Sharp** [comment]
> Is any official task for publishing into Microsoft Store on Azure DevOps and if no, will you consider one? On Azure DevOps is one, which is 2 years without commit and last release according to GitHub is 4 years old.

@JanRajnoha we use this one
https://marketplace.visualstudio.com/items?itemName=MS-RDX-MRO.windows-store-publish

**Comment by JanRajnoha** [comment]
@Panda-Sharp Yeah, I know about it and probably it still works. I just hoped for some updated tool, but I will try old one

**Comment by Panda-Sharp** [comment]
> @Panda-Sharp Yeah, I know about it and probably it still works. I just hoped for some updated tool, but I will try old one

@JanRajnoha still works for me 🙂
